From 2975e4c2aa079835a972dced6a90b1f8c471beaa Mon Sep 17 00:00:00 2001
From: Davide Depau <davide@depau.eu>
Date: Sun, 10 Jan 2021 16:54:37 +0100
Subject: [PATCH] arm64: dts: rockchip: Disable eMMC HS400-ES for Orange Pi 4

It doesn't seem to work very stably on at least one of the board I own.

HS400 without enhanced strobe seems to work stably enough.
---
 arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts b/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts
index 2ad00db1ed87..b6f8b3ca90bf 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-orangepi-4.dts
@@ -894,7 +894,7 @@ &sdhci {
 	bus-width = <8>;
 	mmc-hs200-1_8v;
 	mmc-hs400-1_8v;
-	mmc-hs400-enhanced-strobe;
+	//mmc-hs400-enhanced-strobe;
 	//supports-emmc;
 	non-removable;
 	//keep-power-in-suspend;
-- 
2.30.0

